### DISCUSSION TITLE
Is opengamepanel.org under maintenance?

### DISCUSSION THREAD

**Discussion by DieFeM** [OpenGamePanel/OGP-Website#618] [General]
OGP website domain opengamepanel.org doesn't resolve to any IP, is it under maintenance or the site just got to its end.
Having in to account that @own3mall hosted it until now, I mention him in hope of getting an answer.

**Comment by Zorrototo** [comment]
There is an issue with the datacenter hosting the website. No ETA on resolution of the issue from what I know.

**Comment by DieFeM** [comment]
It comeback to life already.
